doc/dev/mgmt/tests.md

```diff
@@ -195,11 +167,9 @@ There are also legacy tests in the following three locations:
 
 For more information about legacy tests, see [Legacy tests](https://github.com/Azure/azure-sdk-for-python/wiki/Legacy-tests).
 
-## Using the Azure Python SDK test framework
+## Writing management plane test with the the Azure Python SDK test framework
```

Comment:
Probably want to explain the mgmt/data plane split, and why one might chose one or the other.

Reply:
Pragmatically: azure-mgmt-xxxxxxxx  is mgmt, else is data-plane (that's easy to understand)